src/ui/ChatWindow.ts

```diff
@@ -83,7 +84,65 @@ export default class ChatWindow {
       this.element.attr('data-presence', Presence[this.contact.getPresence()]);
       this.element.attr('data-subscription', this.contact.getSubscription());
 
-      this.getAccount().triggerChatWindowInitializedHook(this, contact);
+      this.getAccount().triggerChatWindowInitializedHook(this, contact);   this.element.find('div.jsxc-message-area').on('contextmenu',{ref:this},this.openContextMenu);
+
+      $(document).on('mousedown', function (e) {
+          // If the clicked element is not the menu
+          // Hide it
+          if (!($(e.target).parents('.jsxc-custom-menu').length > 0)) {
+            $('.jsxc-custom-menu').hide(100);
```

Comment:
Would hide all menus on the page.

Reply:
The intention is to close the context menu if the user clicks outside of it